src/usr/bin/google_set_hostname

```diff
@@ -15,6 +15,25 @@
 
 # Deal with a new hostname assignment.
 
+retain_legacy_behavior() {
+	ggactl=$(which ggactl 2> /dev/null)
+	if [ -z "$ggactl" ]; then
+		# Agent doesn't have ggactl for communication, legacy behavior.
+		return 0
+		old_hostname=$($ggactl)
```

Comment:
do you really want this `old_hostname` attribution here?

Reply:
No, it's not even reachable.